lib/features/thread/presentation/widgets/email_tile_web_builder.dart

```diff
@@ -539,6 +539,16 @@ class EmailTileBuilder {
     }
   }
 
+  FontWeight _buildFontForReadEmail() {
+    if (!_presentationEmail.hasRead) return FontWeight.w600;
+    return FontWeight.normal;
+  }
+
+  Color _buildTextColorForReadEmail() {
+    if (_presentationEmail.hasRead) return AppColor.colorContentEmail;
```

Comment:
IMO, you should write as functional programming:
return _presentationEmail .hasRead ? AppColor.colorContentEmail : AppColor.colorContentEmail;